xmtp/migrations/2023-06_05-134021_accounts/up.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS accounts (
+    id INTEGER PRIMARY KEY NOT NULL,
+    created_at INTEGER NOT NULL,
+    serialized_key TEXT NOT NULL
```

Comment:
I've been storing serialized sessions as blobs. I can change to use TEXT for consistency's sake. It shouldn't really matter one way or the other.

Reply:
I don't have a strong preference, though blob makes more sense. In diesel the difference is whether you get a string or a Vec<u8>. I'll change

Reply:
@neekolas what binary format are you using for serde? CBOR?

Reply:
I've just been naively using `serde_json::to_vec`, but I could be convinced we should standardize on CBOR. Given this database is only meant to be read by a single application, trading performance for compatibility seems reasonable.

Reply:
Awesome thats where I was leaning, but wanted to match. I'll set the tone and we can debate later

Reply:
FYI: I avoided CBOR as serde_cbor is EOL. https://crates.io/crates/serde_cbor

Should probably think a little before jump down that rabbit hole